stdlib/inspect.pyi

```diff
@@ -23,7 +23,7 @@ from types import (
 if sys.version_info >= (3, 7):
     from types import ClassMethodDescriptorType, WrapperDescriptorType, MemberDescriptorType, MethodDescriptorType
 
-from typing import Any, ClassVar, NamedTuple, Protocol, Tuple, Type, TypeVar, Union
+from typing import AbstractSet, Any, ClassVar, NamedTuple, Protocol, Tuple, Type, TypeVar, Union
```

Comment:
Considering that we prefer to use imports from `collections.abc`, I'd suggest to use `from collections.abc import Set as AbstractSet`. Best of both worlds.

Reply:
Done!